### PR TITLE
Update pom.xml files formatting

### PR DESCRIPTION
**Pull request by davorbonaci**
This makes it easier to run maven-release-plugin and understand the output. Otherwise, the release plugin will make such changes repeatedly in a release branch.

### CODE REVIEW

**Comment by davorbonaci** [comment]
R: @dhalperi 
CC: @jbonofre

File: pom.xml [hunk 1]
```diff
@@ -97,7 +95,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <dataflow.javadoc_opts></dataflow.javadoc_opts>
+    <dataflow.javadoc_opts />
```
Comment (dhalperi) [inline pom.xml]: space necessary?

File: runners/flink/examples/pom.xml [hunk 2]
```diff
@@ -107,7 +105,7 @@
           <executable>java</executable>
           <arguments>
             <argument>-classpath</argument>
-            <classpath/>
+            <classpath />
```
Comment (dhalperi) [inline runners/flink/examples/pom.xml]: space necessary?

File: runners/spark/pom.xml [hunk 3]
```diff
@@ -193,8 +191,7 @@
                 <shadedArtifactAttached>true</shadedArtifactAttached>
                 <shadedClassifierName>spark-app</shadedClassifierName>
                 <transformers>
-                  <transformer
-                    implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+                  <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
```
Comment (dhalperi) [inline runners/spark/pom.xml]: space necessary?

**Comment by dhalperi** [comment]
LGTM, one nit 3x.

**Comment by jbonofre** [comment]
LGTM (sorry for the delay)

**Pull request closed without merging**